README.md

```diff
@@ -85,9 +85,23 @@ Use [`codespell`](https://github.com/codespell-project/codespell) to check for c
   nox -s spelling -- -w -i 3
   ```
 
+## Dependency files
+
+`nox` sessions use dependencies from requirements files in the `tests/` directory.
+Each session has a `tests/{name}.in` file with *direct* dependencies and a lock file in `tests/{name}.txt` that specifies exact versions for *direct* and *transitive* dependencies.
```

Comment:
I changed what was italicized now that you've reworded this. The main part is the exact versions.

```suggestion
Each session has a `tests/{name}.in` file with direct dependencies and a lock file in `tests/{name}.txt` that pins *exact versions* for both direct and transitive dependencies.
```